content/en/docs/CRDs/_index.md

```diff
@@ -1,11 +1,21 @@
 ---
-title: "Custom Resource Definitions"
+title: "Resource Definitions"
 weight: 80
 description: >
   Custom Resource Definitions (CRDs) for Kyverno policies and other types.
 ---
 
-Kyverno uses Kubernetes Custom Resource Definitions (CRDs) for policy definitions, policy reports, and other internal types. When operating in a Kubernetes cluster with Kyverno installed, you can always inspect Kyverno types natively using `kubectl explain`. 
+Kyverno uses Kubernetes Custom Resource Definitions (CRDs) for policy definitions, policy reports, and other internal types.
+
+The complete Kyverno CRD reference can be viewed using this link:
+
+> **[kyverno.io/v1.ClusterPolicy](https://htmlpreview.github.io/?https://github.com/kyverno/kyverno/blob/main/docs/crd/v1/index.html#kyverno.io/v1.ClusterPolicy)**
+
+The HTML source is available in the [Kyverno GitHub repository](https://github.com/kyverno/kyverno/tree/main/docs) and generated from type definitions stored at [kyverno/kyverno/pkg/api](https://github.com/kyverno/kyverno/tree/main/pkg/api).
+
+## kubectl explain 
+
+When operating in a Kubernetes cluster with Kyverno installed, you can always inspect Kyverno types natively using `kubectl explain`. 
```

Comment:
Does it make sense to write a small note about how this also enables IDE integrations, ex., VS Code? If not here, maybe elsewhere? It's a very nice feature and many may not know the two pieces of resulting functionality come from the same thing.

Reply:
Yes, I agree. Perhaps we can pull the `kubectl explain` and VS Code integration sections into Writing policies or a new `Validating Policies` section? We also have community contributions for GitHub precommit actions to help validate policies, we can also reference that.

Reply:
I will log a separate issue to track that.

Reply:
Logged: https://github.com/kyverno/website/issues/135